apps/src/templates/sectionProgress/standards/StandardsIntroDialog.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Portal from 'react-portal';
+import i18n from '@cdo/locale';
+import Dialog, {Title, Body} from '@cdo/apps/templates/Dialog';
+import color from '@cdo/apps/util/color';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+const styles = {
+  description: {
+    color: color.dark_charcoal
+  },
+  boldText: {
+    fontFamily: '"Gotham 7r", sans-serif'
+  }
+};
+
+export class StandardsIntroDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    handleConfirm: PropTypes.func.isRequired,
+    handleCancel: PropTypes.func.isRequired
+  };
+
+  render() {
+    return (
+      <Dialog
+        isOpen={this.props.isOpen}
+        confirmText={i18n.gotIt()}
+        onConfirm={this.props.handleConfirm}
+      >
+        <Title>{i18n.progressOnCSTAStandards()}</Title>
+        <Body>
+          <div style={styles.description}>
+            <p>
+              {i18n.progressOnCSTAStandardsDescription()}{' '}
+              <a
+                href="https://www.csteachers.org/page/standards"
+                target="_blank"
+                style={styles.boldText}
+              >
+                {i18n.CSTAStandards()}
```

Comment:
I want to suggest that you combine the two messages into one, and then pass in the URL, to make this more translatable, like this:
https://github.com/code-dot-org/code-dot-org/blob/fdb73008ec0aa8edf3f6393eaff5da7eb6c0e049/apps/i18n/common/en_us.json#L833 https://github.com/code-dot-org/code-dot-org/blob/ce87cd73a12fca5bf094c808be3d0598969c78ca/apps/src/templates/teacherDashboard/SectionLoginInfo.jsx#L267-L268

However, I don't know how to do that with the `target="_blank"` property. @Hamms , what is the best practice here?